editoast/editoast_common/src/tracing.rs

```diff
@@ -59,20 +53,11 @@ pub fn create_tracing_subscriber<T: SpanExporter + 'static>(
     let telemetry_layer = match tracing_config.telemetry {
         None => None,
         Some(telemetry) => {
-            let resource = Resource::new(vec![KeyValue::new(
-                opentelemetry_semantic_conventions::resource::SERVICE_NAME,
-                telemetry.service_name.clone(),
-            )])
-            .merge(&Resource::from_detectors(
-                Duration::from_secs(10),
-                vec![
-                    Box::new(SdkProvidedResourceDetector),
-                    Box::new(TelemetryResourceDetector),
-                    Box::new(EnvResourceDetector::new()),
-                ],
-            ));
-            let otlp_tracer = opentelemetry_sdk::trace::TracerProvider::builder()
-                .with_batch_exporter(exporter, opentelemetry_sdk::runtime::Tokio)
+            let resource = Resource::builder()
```

Comment:
Just a note, the `builder` automatically configure the 3 resource detectors that were part of the previous code, hence, no need for these anymore.